lib/metadata.js

```diff
@@ -898,4 +1099,115 @@ function adaptKeyTypes(typesString, encoder) {
   return types;
 }
 
+/**
+ * Creates a new SchemaFunction.
+ * @class
+ * @classdesc Describes a CQL function.
+ * @param {Encoder} encoder
+ * @param {Row} row
+ * @constructor
+ */
+function SchemaFunction(encoder, row) {
```

Comment:
It could be helpful to also have a field for the associated keyspace for both SchemaFunction and SchemaAggregate as that may be required depending on whether or not the Client has a set keyspace.